deploy-disconnected-registry/olm-sync.sh

```diff
@@ -74,11 +108,16 @@ function mirror() {
     registry_login ${SOURCE_REGISTRY}
     echo ">>>> Podman Login into Destination Registry: ${DESTINATION_REGISTRY}"
     registry_login ${DESTINATION_REGISTRY}
+
     [ -d ~/.docker ] || mkdir  ~/.docker
+    sleep 15
+    echo ">>>> Copy pull secret to ~/.docker/config.json"
+    mkdir -p ~/.docker
     cp -f ${PULL_SECRET} ~/.docker/config.json
-
     mkdir -p /var/run/user/0/containers
     cp -f ${OUTPUTDIR}/pull-secret.json /var/run/user/0/containers/auth.json
+    cp -f ${PULL_SECRET} /var/run/user/0/containers/auth.json
```

Comment:
This overwrites the previous copy making it redundant.

Reply:
Love that you noticed our "desperate double copy" line. :D